partner_contact_tag/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Partner Contact Tag",
+    "summary": """
+        Display tag in the pop-up form view when adding a contact to a partner.""",
+    "version": "16.0.1.0.0",
+    "category": "Uncategorized",
+    "website": "https://github.com/coopiteasy/addons",
+    "author": "Coop IT Easy SC",
+    "maintainers": ["carmenbianca"],
+    "license": "AGPL-3",
+    "application": False,
+    "depends": [
+        "base",
+    ],
+    "excludes": [],
+    "data": [
+        "views/res_partner_views.xml",
+    ],
+    "demo": [],
+    "qweb": [],
```

Comment:
```suggestion
    "depends": [
        "base",
    ],
    "data": [
        "views/res_partner_views.xml",
    ],
```
please [avoid empty keys](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id6).